src/LIBSVM.jl

```diff
@@ -368,6 +368,13 @@ function svmtrain(
               (Ptr{Cvoid},), @cfunction(svmnoprint, Cvoid, (Ptr{UInt8},)))
     end
 
+    err = ccall((:svm_check_parameter, libsvm), Cstring,
+                (Ptr{SVMProblem}, Ptr{SVMParameter}),
+                problem, param)
+    if err != C_NULL
+        throw(ArgumentError("Incorrect parameter: $(unsafe_string(err))"))
+    end
+
```

Comment:
Could you make this code block as a function?

Seem `svm_cross_validation` can reuse it (, although we don't provide that function binding ATM)

https://github.com/cjlin1/libsvm/blob/eedbb147ea79af44f2ecdca1064f2c6a8fe6462d/README#L613-L619